docs/VARIABLES-MAPPING.md

```
@@ -1,188 +1,188 @@
+| `--shadow-md` | `var(--shadow-md)` |
+| `--shadow-lg` | `var(--shadow-lg)` |
+| `--shadow-xl` | `var(--shadow-xl)` |
+| `--shadow-2xl` | `var(--shadow-2xl)` |
+
+Usage: `shadow-sm`, `shadow`, `shadow-md`, etc.
+
+---
+
+## Flow summary
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)